doozer/doozerlib/cli/images_streams.py

```diff
@@ -748,6 +748,10 @@ def search_issues(query):
             issue = jira_client.create_issue(
                 fields
             )
+            # check depend issues and set depend to a higher version issue if ture
+            depend_issues = search_issues(f"Update {major}.{minor+1} {image_meta.name} image to be consistent with ART")
```

Comment:
Shouldn't this be a jql? Did you test this?

Reply:
updated, test with
```
> jira_client.search_issues("project=OCPBUGS AND summary ~ 'Update 4.14 ose-apiserver-network-proxy image to be consistent with ART'")
[<JIRA Issue: key='OCPBUGS-10187', id='15143318'>]
```